CanvasPlusPlayground/Views/CourseListView.swift

```diff
@@ -43,20 +43,29 @@ struct CourseListView: View {
     }
 
     private var mainBody: some View {
-        List(courseManager.courses, id: \.id) { course in
-            NavigationLink(course.name ?? "", value: course)
-        }
-        .navigationTitle("Courses")
-        .toolbar {
-            ToolbarItem(placement: .topBarLeading) {
-                Button("Change Access Token", systemImage: "gear") {
-                    showSheet.toggle()
+            ScrollView {
```

Comment:
You can keep this as a `List` and wrap the `ForEach`’s you have below in `Section`’s.